packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -429,7 +429,7 @@ class _DialPainter extends CustomPainter {
       center: focusedPoint, radius: focusedRadius
     );
     canvas
-      ..saveLayer(focusedRect, new Paint())
+      ..save()
```

Comment:
Assuming I was looking at the right part of the dial, it didn't seem to make any difference.